includes/models/model.llms.order.php

```diff
@@ -238,13 +238,13 @@ protected function after_create() {
 	}
 
 	/**
-	 * Calculate the date when billing should be
+	 * Calculate the date when a recurring order should stop being billed
 	 *
 	 * Applicable to orders created from plans with a set # of billing intervals.
 	 *
 	 * @since 3.10.0
 	 *
-	 * @return int
+	 * @return int Calculated billing end date as a Unix timestamp.
 	 */
 	private function calculate_billing_end_date() {
```

Comment:
We don't use this anymore, and ti returns something unreliable, because the "start" is not correct.
RIght?
So we can remove this.

Reply:
@eri-trabiccolo yes you're right!